app/views/admin/customers/index.html.haml

```diff
@@ -61,36 +61,37 @@
             -# %th.bulk
               -# %input{ :type => "checkbox", :name => 'toggle_bulk', 'ng-click' => 'toggleAllCheckboxes()', 'ng-checked' => "allBoxesChecked()" }
             %th.email{ 'ng-show' => 'columns.email.visible' }
-              %a{ :href => '', 'ng-click' => "predicate = 'customer.email'; reverse = !reverse" }=t('admin.email')
+              %a{ :href => '', 'ng-click' => "sorting.predicate = 'customer.email'; sorting.reverse = !sorting.reverse" }=t('admin.email')
```

Comment:
Setting `reverse` to `true` initially is a lot better than `false`. But it's still a bit weird. If I click on Email, it's sorting normally. If I then click on another column, it's reversed, right? I'm wondering if we should add the method `sorting.sort(column)` with code like:
```coffee
reverse = (predicate == column) && !reverse
predicate = column
```

Reply:
@mkllnk Yes, you're right - it should reset the reverse setting when switching to a new column. I will make a follow-up change for that.